x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/documents.tsx

```diff
@@ -74,7 +75,7 @@ export const SearchIndexDocuments: React.FC = () => {
 
   const { makeRequest: getDocuments } = useActions(documentLogic);
   const { makeRequest: getMappings } = useActions(mappingLogic);
-  const { data, status } = useValues(documentLogic);
+  const { data, status, error } = useValues(documentLogic);
```

Comment:
We should probably add a callout for when there's an error but the error is not 404 (or the error occurs on an index that's not the access control switcher).

Reply:
Currently the regular flash message is shown. It would be simple to add a new one for error cases.

Reply:
Cool cool, as long as the page doesn't break that's good enough for me.